cypress-slack-reporter-old/package.json

```diff
@@ -21,7 +21,7 @@
     "type": "git",
     "url": "git+https://github.com/you54f/cypress-slack-reporter.git"
   },
-  "author": "Yousaf Nabi",
```

Comment:
could you revert the author change please

you can use the contributor array for attribution :)

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#people-fields-author-contributors

also in the readme there is a facewall